admin-dev/themes/new-theme/template/page_header_toolbar.tpl

```diff
@@ -35,15 +35,31 @@
       {foreach from=$toolbar_btn item=btn key=k}
         {if $k != 'back' && $k != 'modules-list'}
           {* TODO: REFACTOR ALL THIS THINGS *}
-          <a id="page-header-desc-{$table}-{if isset($btn.imgclass)}{$btn.imgclass|escape}{else}{$k}{/if}" class="toolbar-button toolbar_btn {if isset($btn.target) && $btn.target} _blank{/if} pointer"{if isset($btn.href)} href="{$btn.href|escape}"{/if} title="{if isset($btn.help)}{$btn.help}{else}{$btn.desc|escape}{/if}"{if isset($btn.js) && $btn.js} onclick="{$btn.js}"{/if}{if isset($btn.modal_target) && $btn.modal_target} data-target="{$btn.modal_target}" data-toggle="modal"{/if}{if isset($btn.help)} data-toggle="tooltip" data-placement="bottom"{/if}>
+          <a
+            class="m-b-2 m-r-1 btn btn-primary {if isset($btn.target) && $btn.target} _blank{/if} pointer"{if isset($btn.href)}
+            id="page-header-desc-{$table}-{if isset($btn.imgclass)}{$btn.imgclass|escape}{else}{$k}{/if}"
+            href="{$btn.href|escape}"{/if}
+            title="{if isset($btn.help)}{$btn.help}{else}{$btn.desc|escape}{/if}"{if isset($btn.js) && $btn.js}
+            onclick="{$btn.js}"{/if}{if isset($btn.modal_target) && $btn.modal_target}
+            data-target="{$btn.modal_target}"
+            data-toggle="modal"{/if}{if isset($btn.help)}
+            data-toggle="tooltip"
+            data-placement="bottom"{/if}
+          >
             <i class="material-icons">{$btn.icon}</i>
             <span class="title">{$btn.desc|escape}</span>
           </a>
         {/if}
       {/foreach}
       {if isset($toolbar_btn['modules-list'])}
         {* TODO: REFACTOR ALL THIS THINGS *}
```

Comment:
We should have an URL instead:

![REFACTOR ALL THE THINGS](http://devops.com/wp-content/uploads/2014/08/refactor-all-the-things.jpg)